config

```diff
@@ -899,6 +899,12 @@ fi
 OUT="$OUT"
 
 $PERL $THERE/Configure LIST | grep "$OUT" > /dev/null
```

Comment:
What's the exit code of "foo | bar" ?  Is it foo or bar?  Answer is it depends on the shell.  Perhaps rewrite this into two parts and explicitly save the command status that you care about.  Also, the if/echo statements on the following line will clobber the exit status anyway; try this:
```
MAC ~$ cat test 
false
echo $?
if [ "$1" = "y" ] ; then echo ok; fi
echo $?

MAC ~$ sh test y
1
ok
0

MAC ~$ sh test
1
0

MAC ~$ 
```

Reply:
Ah ! Good catch